llama-challenge/llama-challenge-starter-kit.ipynb

```diff
@@ -46,6 +46,8 @@
    "id": "9830e7a2",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #27.        target_audio_embedding = audio_model(audio.unsqueeze(0))
</pre><p>Should you not also rather do batched audio (example comment in review)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ByteFuse/indabax-2021/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>